### PR TITLE
Add sign to numpy_func.py

### LINKED ISSUE #1174: Error when using function numpy.sign
I have a TypeError when I execute this portion of code:

import pint
import numpy as np

ureg = pint.UnitRegistry()
a = -2*ureg.m
np.sign(a)

TypeError: operand type(s) all returned NotImplemented from __array_ufunc__(<ufunc 'sign'>, '__call__', <Quantity(-2, 'meter')>): 'Quantity'

I use version 1.19.2 of numpy and 1.16 of Pint. 

Do I miss something ? 

Regards,

Robin Tesse

### PR DESCRIPTION
**Pull request by rtesse**
- [x] Closes #1174 
- [x] Executed ``black -t py36 . && isort -rc . && flake8`` with no errors
- [x] The change is fully covered by automated unit tests
- [x] Documented in docs/ as appropriate
- [ ] Added an entry to the CHANGES file

### CODE REVIEW

File: docs/numpy.ipynb [hunk 1]
```diff
@@ -203,7 +203,7 @@
     "- **Math operations**: `add`, `subtract`, `multiply`, `divide`, `logaddexp`, `logaddexp2`, `true_divide`, `floor_divide`, `negative`, `remainder`, `mod`, `fmod`, `absolute`, `rint`, `sign`, `conj`, `exp`, `exp2`, `log`, `log2`, `log10`, `expm1`, `log1p`, `sqrt`, `square`, `cbrt`, `reciprocal`\n",
     "- **Trigonometric functions**: `sin`, `cos`, `tan`, `arcsin`, `arccos`, `arctan`, `arctan2`, `hypot`, `sinh`, `cosh`, `tanh`, `arcsinh`, `arccosh`, `arctanh`\n",
     "- **Comparison functions**: `greater`, `greater_equal`, `less`, `less_equal`, `not_equal`, `equal`\n",
-    "- **Floating functions**: `isreal`, `iscomplex`, `isfinite`, `isinf`, `isnan`, `signbit`, `copysign`, `nextafter`, `modf`, `ldexp`, `frexp`, `fmod`, `floor`, `ceil`, `trunc`\n",
+    "- **Floating functions**: `isreal`, `iscomplex`, `isfinite`, `isinf`, `isnan`, `signbit`, `signbit`, `copysign`, `nextafter`, `modf`, `ldexp`, `frexp`, `fmod`, `floor`, `ceil`, `trunc`\n",
```
Comment (dalito) [inline docs/numpy.ipynb:206]: Copy paste error? 2 x signbit

Comment (rtesse) [reply docs/numpy.ipynb]: Yes sorry, I just pushed the corrected file.

**Comment by hgrecco** [comment]
Let me know when it is ready for review

**Comment by rtesse** [comment]
I have committed all the files so normally it is ok for review.

**Pull request merged**